css/properties/vector-effect.json

```
@@ -10,12 +10,12 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "≤80"
+              "version_added": "≤15"
             },
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "≤72"
+              "version_added": "≤50"
```

```suggestion
              "version_added": "15"
```

See: https://github.com/caugner/css-properties/commit/14f2c5c997fbc959f4fdb8e6dc690267ca4b35eb#diff-b63d72471087ee495d4f5c7e94b58bef200825135bdee1bdcddbfbd594b0c328R194